README.md

```diff
@@ -3,21 +3,37 @@
 Official frontend for [libSMCE](https://github.com/ItJustWorksTM/libSMCE) using [Godot](https://godotengine.org/).  
 Initially created to emulate cars supporting the [smartcar_shield](https://github.com/platisd/smartcar_shield) platform.
 
-### Resources
-
-* [Releases](https://github.com/ItJustWorksTM/smce-gd/releases)
-* [Setup](https://github.com/ItJustWorksTM/smce-gd/wiki)
-
 ### Dependencies
 
 * _[Godot](https://godotengine.org)_
-* *_[libSMCE](https://github.com/ItJustWorksTM/libSMCE)_ ([version]([./CMakeLists.txt#L28](https://github.com/ItJustWorksTM/smce-gd/blob/master/CMakeLists.txt#L28)))
+* *_[libSMCE](https://github.com/ItJustWorksTM/libSMCE)_ ([version](https://github.com/ItJustWorksTM/smce-gd/blob/master/CMakeLists.txt#L28))
 * _[godot-cpp](https://github.com/godotengine/godot-cpp)_ (automatically built from source; *_SConstruct_ is **not** used, but _Python3_ is still required)
 * C++20-compatible compiler + _[CMake](https://cmake.org)_
 
-\* To install libSMCE head to it's [releases](https://github.com/ItJustWorksTM/libSMCE/releases) page and extract/install one of the artifacts, then set the env var `SMCE_ROOT` pointed to the root of the extracted directory.
+### Prerequisites
+
+1. **Setup** (follow the setup for: [Windows](https://github.com/ItJustWorksTM/smce-gd/wiki/Windows-setup) / [MacOS](https://github.com/ItJustWorksTM/smce-gd/wiki/MacOS-setup) / [Debian-based GNU/Linux](https://github.com/ItJustWorksTM/smce-gd/wiki/Debian-based-Linux-setup))
+2. [libSMCE Releases](https://github.com/ItJustWorksTM/libSMCE/releases) (extract/install the artifact for your OS, Windows users should get the *Release* version)
```

Comment:
"Windows users should get the *Release* version" This version does not exist in 1.3.2